packages/core/src/text/Action.tsx

```diff
@@ -0,0 +1,22 @@
+import { forwardRef } from "react";
+import { makePrefixer } from "../utils";
+import { clsx } from "clsx";
+import { Text, TextProps } from "./Text";
+
+const withBaseName = makePrefixer("saltText");
+
+export const Action = forwardRef<
```

Comment:
this and Notation components aren't really theme updates. I would leave them out of this PR, only adding tokens, and then put these a new pr for new Text component variants